klippy/extras/probe.py

```diff
@@ -346,18 +360,21 @@ def start_probe(self, params):
             self.probe_offsets = (0., 0., 0.)
             self._manual_probe_start()
             return
+        else:
+            self.lift_speed = probe.lift_speed
         # Perform automatic probing
         self.lift_speed = probe.get_lift_speed(params)
```

Comment:
Similar for these two lines.  They also look redundant.

Reply:
You are an eagle.

Reply:
Obsoleted by refactoring